schematic/utils/curie_utils.py

```diff
@@ -6,7 +12,7 @@ def extract_name_from_uri_or_curie(item):
     elif len(item.split("//")[-1].split('/')) > 1:
         return item.split("//")[-1].split('/')[-1]
     else:
-        print("error")
+        logger.error("Error extracting name from URI or Curie.")
```

Comment:
Shouldn't this be raising an exception?

Reply:
yes, many a print statement currently tuck exceptions under the rug; let's raise an exception here and keep the logger output as well.

Reply:
Good catch! Yes, I'm raising a `ValueError` now instead of simply logging an error message.